Documentation/ApiOverview/Logging/Writers/Index.rst

```diff
@@ -167,6 +167,48 @@ Only in the case that all registered writers fail, the log entry plus additional
 will be added to the configured fallback logger (which defaults to
 the :ref:`PhpErrorLog <logging-writers-php>` writer).
 
+.. _logging-writers-usage:
+
+Usage in Custom Class
+---------------------
+
+All log writers can be used in your own classes. You can initialize the loggers in the __construct method of your class :code:`\MyDomain\MyExtension\MyFolder\MyClass`.
+
+.. code-block:: php
+
+    namespace MyDomain\MyExtension\MyFolder;
+
+    use TYPO3\CMS\Core\Utility\GeneralUtility;
+    use TYPO3\CMS\Core\Log\LogManager;
```

Comment:
You should not use this anymore:

https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/Logging/Quickstart/Index.html#instantiate-a-logger-for-the-current-class